src/main/scala/jigg/pipeline/RegexDocumentAnnotator.scala

```diff
@@ -0,0 +1,29 @@
+package jigg.pipeline
+
+import java.util.Properties
+import scala.xml.Node
+
+class RegexDocumentAnnotator(override val name: String, override val props: Properties) extends Annotator {
+  private[this] var documentID: Int = 0
```

Comment:
Use instead `jigg.util.IDGenerator` to manage documentID. See `RegexSentenceAnnotator` for example. Incrementing var value is not thread-safe.

```Scala
val documentIDGen = jigg.util.IDGenerator("d")
…
  <docment id={docmentIDGen.next} … >
```